### PR TITLE
Update comments and examples to use Wazuh instead of OSSEC

### PR DESCRIPTION
**Pull request by palaciosjeremias**
|Related issue|
|---|
|#786|

#### Description
This PR update comments and log examples to use **Wazuh** name instead of **OSSEC**.
It updates every file that doesn't require a rule modification.
Updates examples, comments, paths, puppet modules. It doesn't change recognitions nor license. References to specific files like **ossec-authd** or **ossec.conf** are conserved to be replaced once the file name changes.

#### Tests
- [x] Run `/var/ossec/bin/ossec-logtest -v` with every modified example and verify it's correctly handled.

### CODE REVIEW

File: decoders/0320-sudo_decoders.xml [hunk 1]
```diff
@@ -10,9 +10,9 @@
 <!--
   -  Will extract the username
   -  Examples:
-  -  Apr 27 15:22:23 niban sudo:     dcid : TTY=pts/4 ; PWD=/home/dcid ; USER=root ; COMMAND=/usr/bin/tail /var/log/snort/alert.fast
-  -  Apr 14 10:59:01 enigma sudo:     dcid : TTY=ttyp3 ; PWD=/home/dcid/ossec-hids.0.1a/src/analysisd ; USER=root ; COMMAND=/bin/cp -pr ../../bin/addagent ../../bin/osaudit-logaudit ../../bin/ossec-execd ../../bin/ossec-logcollector ../../bin/ossec-maild ../../bin/ossec-remoted /var/ossec/bin
-  -  Apr 19 14:52:02 enigma sudo:     dcid : TTY=ttyp3 ; PWD=/var/www/alex ; USER=root ; COMMAND=/sbin/chown dcid.dcid .
+  -  Apr 27 15:22:23 niban sudo:     wazuh : TTY=pts/4 ; PWD=/home/wazuh ; USER=root ; COMMAND=/usr/bin/tail /var/log/snort/alert.fast
+  -  Apr 14 10:59:01 enigma sudo:    wazuh : TTY=ttyp3 ; PWD=/home/wazuh/src/analysisd ; USER=root ; COMMAND=/bin/cp -pr ../../bin/addagent ../../bin/osaudit-logaudit ../../bin/ossec-execd ../../bin/ossec-logcollector ../../bin/ossec-maild ../../bin/ossec-remoted /var/ossec/bin
+  -  Apr 19 14:52:02 enigma sudo:    wazuh : TTY=ttyp3 ; PWD=/var/www/alex ; USER=root ; COMMAND=/sbin/chown dcid.dcid .
```
Comment (pereyra-m) [inline decoders/0320-sudo_decoders.xml:15]: The _chown_ command may replace the **dcid** user just to be consistent.

File: rules/0340-puppet_rules.xml [hunk 2]
```diff
@@ -53,7 +53,7 @@
 
     <!--
     Permission
-    - Oct  1 17:56:52 puppet puppet-master[7879]: Permission denied - /etc/puppet/modules/ossec/files/ossec-agent-win32-2.8.exe
+    - Oct  1 17:56:52 puppet puppet-master[7879]: Permission denied - /etc/puppet/modules/wazuh/wazuh-agent-4.1.0-1.msi
```
Comment (pereyra-m) [inline rules/0340-puppet_rules.xml:56]: The _files_ word is removed from the path. Was it intentional ?

**Review by pereyra-m — APPROVED** [review]
LGTM!

File: rules/0340-puppet_rules.xml (hunk 2 above)
Comment (palaciosjeremias) [reply rules/0340-puppet_rules.xml:56]: Yes. The path where the installer will be placed is customizable, but by default, Wazuh Puppet module places it there.

File: decoders/0320-sudo_decoders.xml (hunk 1 above)
Comment (palaciosjeremias) [reply decoders/0320-sudo_decoders.xml]: You are right! Fixed!

**Review by DProvinciani — APPROVED** [review]
Looks good!

File: rules/0340-puppet_rules.xml (hunk 2 above)
Comment (pereyra-m) [reply rules/0340-puppet_rules.xml:56]: Excellent!

**Pull request merged**